papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraOAuthTokenFactory.java

```diff
@@ -0,0 +1,56 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthRsaSigner;
+import com.google.api.client.http.apache.ApacheHttpTransport;
+import com.google.api.client.util.Base64;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+
+class JiraOAuthTokenFactory {
+
+    private final transient String accessTokenUrl;
+    private final transient String requestTokenUrl;
```

Comment:
`final transient`, hmmm jak to ma działać? Tym bardziej, że obiekt nie jest serializowalny.

Reply:
Dodano supress warning, na zasadę, która tego wymagała

Reply:
Na jaką zasadę? Ja się tylko pytam jaki jest sens tego :smile:

Reply:
`Found non-transient, non-static member. Please mark as transient or provide accessors.` Sens tego rule'a?

Reply:
To ten transient jest _trochę_ mylący.

Reply:
Jak to pisałeś to nie zauważyłem, że się jeszcze w jednym miejscu ostał `transient`. Na początku to było na prędce dodane, żeby dla draftu przeszedł build i pierwsza iteracja code review mogła się odbyć